ngraph/core/src/op/mvn.cpp

```diff
@@ -49,6 +49,8 @@ op::MVN::MVN(const Output<Node>& data, AxisSet reduction_axes, bool normalize_va
     , m_reduction_axes{reduction_axes}
 {
     constructor_validate_and_infer_types();
+    const size_t chanelAxis = 1;
```

Comment:
This change I'm not 100% sure of, but it's the only path that we have a problem on:

1. _MVN_ is created by the first creator
2. It's then copied using _clone_with_new_inputs_
3. When running a second creator from _clone_with_new_inputs_ we lose this _m_across_channels_ flag. 

That is why I'm re-checking if it should be enabled.

Reply:
Do we have a test cases which show this issue?

Reply:
Yeah, for example ReadMVNNetwork from IE functional tests was failing.